src/js/AppContainer.jsx

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+var api = require('./api');
+var UserSelect = require('./UserSelect.jsx');
+var UserFiHeading = require('./UserFiHeading.jsx');
+
+var AppContainer = React.createClass({
+
+  getInitialState: function(){
+    return {
+      user: {
+        name: '',
+        institutions: []
+      }
+    }
+  },
+
+  selectCallback: function(e){
+    var user = e.target.value;
+    var self = this;
+    console.log('selecting user: ', user);
+    api.getInstitutions(user, function(institutions){
+      self.setUser({name: user, institutions: institutions});
+    });
+  },
+
+  setUser: function(user){
+    this.setState({user: user});
+  },
+
+  render: function() {
+    return (
+      <div>
+        <UserSelect callback={this.selectCallback}/>
+        <UserFiHeading institution={{}} year="2017" user={this.state.user.name}/>
```

Comment:
Everything works when clicking the "Begin filing now" link to see the upload form except that the heading, `UserFiHeading.jsx` doesn't get updated. Looks like that because it's always getting an empty object, `{}` for `institution`. 

Do we need to add a value to the `state` to hold the institution when one is selected?